docs/network-policy.md

```diff
@@ -295,10 +332,10 @@ Antrea Policy CRDs are ordered based on priorities set at various levels.
 ### Ordering based on Tier priority
 
 With the introduction of tiers, Antrea Policies, like ClusterNetworkPolicies,
-are first enforced based on the tier to which they are associated. i.e. all
-policies belonging to the "Emergency" tier are enforced first, followed by
-policies belonging to the "SecurityOps" tier and so on, until the "Application"
-tier policies are enforced.
+are first enforced based on the Tier to which they are associated. i.e. all
+policies belonging to a higher order Tier are enforced first, followed by
+policies belonging to the next ordered Tier and so on, until the lowest ordered
+"application" Tier policies are enforced.
```

Comment:
you sometimes write "higher order tier" / sometimes "higher ordered tier". I would say the "order" / "ordered" is not needed here: "high Tier", "lowest Tier", "next Tier", etc. convey the meaning pretty well.

Reply:
done